src/browser/core_state.js

```diff
@@ -124,6 +124,8 @@ function appByUuid(uuid) {
     return coreState.apps.find(app => uuid === app.uuid);
 }
 
+const getAppByUuid = appByUuid;
```

Comment:
Discussed offline, we should eventually switch over all usages of `appByUuid` to `getAppByUuid` to better conform to the convention.